### PR TITLE
(PUP-4233, PUP-4234, PUP-4235) Enable Puppet Passenger Acceptance

### PR DESCRIPTION
**Pull request by MikaelSmith**
* (PUP-4233) Add node configs for puppet-passenger
* (PUP-4234) Modify AIO pre-suite to use passenger

### CODE REVIEW

**Comment by MikaelSmith** [comment]
That makes sense, and makes the description in PUP-4233 make sense too.

**Comment by MikaelSmith** [comment]
This works, but I'm still cleaning it up and testing all the new configs.

File: acceptance/config/nodes/debian-7-passenger.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+HOSTS:
+  master:
+    roles:
+    - master
+    platform: debian-wheezy-x86_64
+    hypervisor: vcloud
+    template: debian-7-x86_64
+    passenger: true
```
Comment (MikaelSmith) [inline acceptance/config/nodes/debian-7-passenger.yaml]: Trying to override is_puppetserver here doesn't appear to work, so I'm going to make these generic server configs and do all the passenger vs puppetserver selection in options files.

**Comment by MikaelSmith** [comment]
Ran the configs, they all seem to work and get puppet-master-passenger packages.

**Comment by justinstoller** [comment]
I think that `server` is an ambiguous term for the host files. Not saying that I love `ubuntu-1404-passenger`, but I don't think I would understand why these configs are special from their name if I wasn't involved in these tickets.

**Comment by MikaelSmith** [comment]
Well, there's nothing specific to passenger in those configs. I named them to differentiate varying the server OS instead of the client.

**Comment by joshcooper** [comment]
Justin and I were working on updates to this test so we don't have to create s new rake target

**Comment by MikaelSmith** [comment]
Ah, ok. I played around with it, but felt a new options file was the simplest.

**Comment by joshcooper** [comment]
Superceded by PR #3859

**Pull request closed without merging**